types/timestamp.go

```diff
@@ -31,3 +31,17 @@ func (ts TimestampSlice) Less(i, j int) bool {
 func (ts TimestampSlice) Swap(i, j int) {
 	ts[i], ts[j] = ts[j], ts[i]
 }
+
+// Clock allows clients to retrieve the current time.
+type Clock interface {
```

Comment:
`modules/consensus/consensusset.go` needs this to be exported so that it can declare a `types.Clock` member. Is there a different way of achieving this?